src/vue-tel-input.vue

```diff
@@ -222,7 +226,12 @@ export default {
         .filter(Boolean)
         .map(country => ({ ...country, preferred: true }));
 
-      return [...preferredCountries, ...allCountries];
+      let countryList = allCountries;
```

Comment:
Are lines from 229 -> 232 deprecated? (already being used in `getCountryList()`)

Reply:
@iamstevendao sorry I missed the notification.....